arroyo-api/src/pipelines.rs

```diff
@@ -251,10 +247,14 @@ pub(crate) async fn create_pipeline<'a>(
         )));
     }
 
-    // TODO: this is very hacky
-    let is_preview = req.name.starts_with("preview");
     if is_preview {
         set_parallelism(&mut program, 1);
+        for node in program.graph.node_weights_mut() {
+            // if it is a kafka sink or file sink, switch to null
+            if let Operator::KafkaSink { .. } | Operator::FileSink { .. } = node.operator {
+                node.operator = Operator::NullSink;
```

Comment:
Should this be a web sink? When I try to preview a pipeline with an insert into kafka:

```sql
CREATE TABLE sink (
  count int
)
WITH (
connection = 'local',
topic = 'test',
serialization_mode = 'json');


INSERT INTO 'sink'
select count(*) as count
from copy
group by hop(interval '2 seconds', interval '10 seconds');
```
I get this error:

> Job cotwkgz9 does not have a web sink

Reply:
switched to a web sink.